dascore/proc/align.py

```
@@ -0,0 +1,284 @@
+    start_pad = mode_params["start_offset_samples"]
+    # Create output array
+    out_shape = list(data.shape)
+    out_shape[dim_axis] = output_size
+    output = np.full(out_shape, fill_value, dtype=data.dtype)
+    # Iterate over all shift positions
+    for idx in np.ndindex(shifts.shape):
+        shift = int(adjusted_shifts[idx])
+        # Build slice for this position
+        src_slice = [slice(None)] * data.ndim
+        dst_slice = [slice(None)] * data.ndim
+        for ax, i in zip(coord_axes, idx):
```

Ruff (0.13.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)